src/rust/en.hentai2read/src/helper.rs

```diff
@@ -0,0 +1,71 @@
+use aidoku::{helpers::uri::*, prelude::*, std::String, std::Vec};
+use alloc::{borrow::ToOwned, string::ToString};
+
+pub const BASE_URL: &str = "https://hentai2read.com";
+
+pub fn create_advanced_search_body(
+	manga_title: Option<String>,
+	artist_name: Option<String>,
+	status: Option<i64>,
+	tag_search_mode: Option<String>,
+	include_tags: Option<Vec<i64>>,
+	exclude_tags: Option<Vec<i64>>,
+) -> String {
+	let mut form_body = format!("cmd_wpm_wgt_mng_sch_sbm=Search&txt_wpm_wgt_mng_sch_nme=&cmd_wpm_pag_mng_sch_sbm=&txt_wpm_pag_mng_sch_nme={}&txt_wpm_pag_mng_sch_ats={}&rad_wpm_pag_mng_sch_sts={}&rad_wpm_pag_mng_sch_tag_mde={}",
+		encode_uri(manga_title.unwrap_or_default()),
+        encode_uri(artist_name.unwrap_or_default()),
+        &status.unwrap_or_default(),
+        &tag_search_mode.unwrap_or_default()
```

Comment:
why are you making arguments optional if you just unwrap to default? and when you call the function you're just wrapping all the arguments in `Some()` anyways so it's useless